symphony/bdk/core/auth/auth_session.py

```diff
@@ -22,6 +26,7 @@ def __init__(self, authenticator):
     async def refresh(self):
         """Trigger re-authentication to refresh the tokens.
         """
+        logger.debug("Re-authenticate and try again")
```

Comment:
This could be the first time we log in so this kind of message can be misleading IMHO

Reply:
I changed it to "Authenticate", docstring says re-authentication. Maybe we shouldn't use this function when logging for the first time.